src/System.Drawing.Common/tests/IconTests.cs

```diff
@@ -385,7 +385,37 @@ public void XpIcon_ToBitmap_Success(int size)
         [ConditionalFact(Helpers.IsDrawingSupported)]
         public void ExtractAssociatedIcon_FilePath_Success()
         {
-            using (Icon icon = Icon.ExtractAssociatedIcon(Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico")))
+            ExtractAssociatedIcon_FilePath_Success(Helpers.GetTestBitmapPath("48x48_multiple_entries_4bit.ico"));
+        }
+
+        [PlatformSpecific(TestPlatforms.Windows)] // UNC
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)] // Fix for #34122
```

Comment:
Nit: I believe the attribute has an optional second string argument that can take the reason instead of it being in a comment.  The advantage of that is it's then available to tooling that looks at the attribute to see why it's being skipped, though I don't know if we take advantage of that anywhere, and we're not currently consistent about using it.

Reply:
done!thank's!